cvprac/cvp_api.py

```diff
@@ -3875,7 +3875,7 @@ def svc_account_token_set(self, username, duration, description):
             return self.clnt.post(url, data=payload)
 
     def svc_account_get_all(self):
-        ''' Get all service account states using Resource APIs.
+        ''' Get all service account config/status states using Resource APIs.
```

Comment:
Since we are switching from the service account Config Resource API to the service account State Resource API can we update this doc string and the get_one below to just say "Get all service accounts state using Resource APIs" and "Get a service account's state using Resource APIs"